plugins/modules/ec2_vpc_subnet.py

```diff
@@ -138,6 +138,11 @@
             returned: I(state=present)
             type: str
             sample: us-east-1a
+        availability_zone_id:
+            description: The AZ ID of the subnet.
+            returned: I(state=present)
```

Comment:
```suggestion
            returned: O(state=present)
```

Reply:
Could you please share the reason behind this suggestion?

Reply:
Semantic markup has changed https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation

Reply:
describes [here](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html) in the section **Semantic markup within module documentation**

Reply:
Agreed! I am curious if there is a guideline that prioritizes semantic markup over macros. However, I think it would be advantageous to incorporate this as a guideline in the cloud content coding standards document.